master/internal/command/summary.go

```diff
@@ -28,6 +28,7 @@ type (
 		ExitStatus     *string                `json:"exit_status"`
 		Misc           map[string]interface{} `json:"misc"`
 		IsReady        bool                   `json:"is_ready"`
+		AgentUserGroup *model.AgentUserGroup  `json:"agent_user_group"`
```

Comment:
This PR does affect the scope of an API call.  @ybt195, please take a look.  The reason that the AgentUserGroup needs to be added is so that the CLI can know what user to try to log in as in the container.